MaraBot/Commands/RaceCommandModule.cs

```diff
@@ -8,33 +10,78 @@ namespace MaraBot.Commands
 {
     using Core;
     using Messages;
-    
+
     public class RaceCommandModule : BaseCommandModule
     {
         public IReadOnlyDictionary<string, Preset> Presets { private get; set; }
+        public IReadOnlyDictionary<string, Option> Options { private get; set; }
 
         [Command("race")]
         [Cooldown(15, 900, CooldownBucketType.User)]
         [RequireGuild]
         public async Task Execute(CommandContext ctx, string presetName)
+        {
+            var success = (presetName == "custom") ?
+                LaunchCustomRace(ctx) :
+                LaunchPresetRace(ctx, presetName);
+
+            var emoji = DiscordEmoji.FromName(ctx.Client, await success ? Display.kValidCommandEmoji : Display.kInvalidCommandEmoji);
+            await ctx.Message.CreateReactionAsync(emoji);
+        }
+
+        public async Task<bool> LaunchCustomRace(CommandContext ctx)
+        {
+            if (ctx.Message.Attachments == null || ctx.Message.Attachments.Count == 0)
```

Comment:
Maybe check for `Count != 1`

Reply:
Fixed in 5842bb79e48080160f9f91c25ff8f32b68e1137e.